andela/bp-esa-frontend
# feat(Filter Tags): Show filter options as tags

**What does this PR do?**

- Adds a bar that shows filter options as tags
- Adds SCSS for the filter bar and tags
- Adds tests

**Description of the task to be completed?**
After selecting a filter a user should be able to see the active filters as tags.

**How should this be manually tested?**

Clone and set up the project using the instruction in the README.
$ git checkout to `ft-see-filter-options-as-tags` branch which contains the changes.
Use $ npm start to start the application
Visit http://localhost:3000/.
On the report page, select a filter you want. A bar will appear containing the various active filters selected.

**What are the relevant pivotal tracker stories?**
[#160437219] - Sub-task

Screenshots:
![screen shot 2018-11-21 at 14 46 21](https://user-images.githubusercontent.com/16057093/48839356-4a2dd980-ed9c-11e8-9fee-27ec438f8cf6.png)
